src/screens/LogIn.js

```diff
@@ -61,6 +64,31 @@ class LogIn extends Component {
 
    }
 
+   resetBtnClick(e){
+      if(this.state.userEmail===""){
```

Comment:
Disregard this statement. I didn't realize it was in the log in screen. Although if we are sending a reset email, it would be better to take them to a new page where they can enter their email only to send the reset password email